src/lambda/expression.rs

```diff
@@ -40,7 +39,7 @@ pub enum Unsafe {
     Option<GroupBy>,
     Option<Arc<DataLoader<HttpDataLoader, NoCache>>>,
   ),
-  JS(Box<Expression>, String),
+  JS(Box<Expression>, JsPluginExecutor),
```

Comment:
```suggestion
  SyncFn(Box<dyn Fn(Context) -> Value> ),
```

We could use SyncFn instead of a specialized JS container. This is more general purpose container representing any computation that can be synchronously completed.